S21/indraneelL/wordConvert.py

```diff
@@ -0,0 +1,29 @@
+import locale
+import re
+
+## read the file and iterate through every line
+class WordConvert:
+    def readTestFile(self):
+        textFile = open("tests.tsv", "r")
+        while textFile:
+            line = textFile.readline()
+            if line == "" or line == '\n':
+                break
+            word, lang, _  = line.split('\t')
```

Comment:
maybe add the 3rd column column instead of using a filler? It makes code readable.